package.json

```diff
@@ -73,9 +75,11 @@
     "glob": "^7.1.3",
     "mocha": "5.2.0",
     "npm-run-all": "^4.1.3",
+    "prettier": "^1.14.3",
```

Comment:
Ooh, I've been bitten by version ranges in auto-formatters before. Please remove the `^`.

Reply:
Sure.